### DISCUSSION TITLE
Como se sube una imagen?

### DISCUSSION THREAD

**Discussion by Karlygs** [matteobruni/tsparticles#5025] [Q&A]
Quiero subir una imagen personalizada en las particulas en multiples imágenes pero si subo una no se ve. Que código tengo que usar para poder usar la imagen que necesito?. gracias

**Comment by matteobruni** [comment]
Can you please use English?

**Comment by DigitalNaut** [comment]
Alguien más compartió este Codepen que usa una imagen. Quizá puedes aprovecharlo para lograr tu objetivo:

https://codepen.io/matteobruni/pen/BaOLEqW
